model/prelude.sail

```diff
@@ -248,6 +248,15 @@ function shift_right_arith32 (v : bits(32), shift : bits(5)) -> bits(32) =
     let v64 : bits(64) = EXTS(v) in
     (v64 >> shift)[31..0]
 
+val ror32 : (bits(32), int) -> bits(32)
```

Comment:
Please use the polymorphic version I provided rather than writing multiple monomorphised copies.

Reply:
Yeah I will, I'm trying to do one thing at a time is all.

Reply:
If it makes it easier for you, feel free to ignore my pushes until I say "hey, I think this is ready" again. I realise it's a bit frustrating otherwise!

Reply:
Oh ok, that would be helpful, yes :)